doc/nomenclature.rst

```diff
@@ -0,0 +1,14 @@
+============
+Nomenclature
+============
+
+Brackets
+--------
+
+ref https://en.wikipedia.org/wiki/Bracket
+
++-------------+----------+--------+
+| ( )         | [ ]      | { }    |
++=============+==========+========+
+| parentheses | brackets | braces |
```

Comment:
I see one problem here: according to the [linked reference](https://en.wikipedia.org/wiki/Bracket) square brackets `[]` are called "brackets" in US english, but the project uses British english, where "brackets" is an alias for parentheses `()`. This seems little inconsistent, but if we treat it as our internal naming convention then it's fine 👍.

PS. is there any reason why we don't cover chevrons `<>` here?

Reply:
It's even more complicated when I read it again:

> There are four primary types of brackets. In British usage they are known as round brackets (or simply brackets), square brackets, curly brackets, and angle brackets; in American usage they are respectively known as parentheses, brackets, braces, and chevrons. There are also various less common symbols considered brackets.

So it seems like we just use American english here on purpose. I am not against it, as I think it's better to use these dedicated names rather than "round|square|curly|angle brackets" for every fixer/option name. I just want to make conscious decision 🙂.